src/model/database.h

```diff
@@ -197,6 +197,7 @@ class ObjectDatabase
   PObjectType GetObjectType(wxString name);
 
   int CountChildrenWithSameType(PObjectBase parent,PObjectType type);
+  int CountChildrenWithTypes(PObjectBase parent,const std::set<wxString> &types);
```

Comment:
I think this really should be an overload instead of another method, also formatting